src/ReleasableImage.cpp

```diff
@@ -0,0 +1,33 @@
+#include <ZividPython/ReleasableImage.h>
+
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
+
+namespace
+{
+    py::buffer_info makeBufferInfo(ZividPython::ReleasableImage &image)
+    {
+        const auto data = image.dataPtr();
+
+        return py::buffer_info{ const_cast<Zivid::RGBA8 *>(data), // TODO: Const casting this until https://github.com/pybind/pybind11/pull/1466 has been merged
```

Comment:
`const_cast` will still be needed after that PR.

https://github.com/pybind/pybind11/issues/1993 will make it possible to remove `const_cast`

Can you test what happens if you try to modify the numpy array and then for example call save on the original image? Do you save the changes or not or does it crash?

Reply:
Calling the following:

```
>>> import zivid
>>> app = zivid.Application()
>>> cam = app.connect_camera()
>>> fra2 = cam.capture_2d(zivid.Settings2d())
>>> im = fra2.image()
>>> im.save("orig.png")
>>> arr = im.to_array()
>>> for i in range(400,500):
...     for j in range (400, 500):
...         arr[i,j][1] = 0
...         arr[i,j][2] = 0
...         arr[i,j][3] = 0
... 
>>> im.save("new.png")
```

This gives me two images that looks exactly the same.